pyproject.toml

```diff
@@ -13,7 +13,7 @@ requires-python = ">= 3.10"
 dependencies = [
     "ctakesclient >= 5.1, < 6",
     "cumulus-fhir-support >= 1.2, < 2",
-    "delta-spark >= 3, < 4",
+    "delta-spark >= 3.2.1, < 4",
```

Comment:
3.2.1 had a specific fix for "clustering by a single column" that we were waiting for, since that's our primary use case here